Source/WebCore/loader/ResourceMonitorChecker.cpp

```diff
@@ -34,8 +34,6 @@
 
 #if ENABLE(CONTENT_EXTENSIONS)
 
-#define RESOURCEMONITOR_RELEASE_LOG(fmt, ...) RELEASE_LOG(ResourceMonitoring, "ResourceMonitorChecker::" fmt, ##__VA_ARGS__)
```

Comment:
Not sure why we're dropping this convenience macro. It would avoid repeating `ResourceMonitorChecker::` every time.